prep-css-syntax/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Syntax</title>
+</head>
+<body>
+  <h1 style="font-family: Arial, Helvetica, sans-serif;">This h1 tag has been given Arial as a font-family.</h1>
+  <p style="font-size: 2rem;">I am a p tag which has a font-size of 2rem!!!</p>
+  <h3 style="color: firebrick;" >This h3 tag has a font color of firebrick.</h3>
+  <h2 style="font-style: italic;">This h2 has an italic font-style.</h2>
```

Comment:
Hi Nhu, for this exercise we want to write our CSS in a `<style></style>` element in the `<head></head>` of the document instead of using a `style` attribute directly on the elements.